docs/examples/index.rst

```diff
@@ -0,0 +1,6 @@
+.. toctree::
+    :maxdepth: 1
+    :caption: examples
+
+    basic_tracer/README
```

Comment:
should we use a broader toctree expression to captures these? IIRC we can use glob. Would save someone having to add to this file every time an example is added.